entries-api/filesystementry-getParent-manual.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Entries API: FileSystemEntry getParent() manual test</title>
+<link rel=help href="https://wicg.github.io/entries-api/#dom-filesystementry-getparentapi-entry">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+
+<script>
+entry_test((t, entry) => {
+  assert_idl_attribute(entry, 'getParent', 'FileSystemEntry has filesystem attribute');
```

Comment:
(I know this is just copied from before) getParent() rather than filesystem, right?

Reply:
D'oh, nice catch. Fixed.